contributors/devel/api-conventions.md

```diff
@@ -1251,29 +1251,38 @@ to, but if unspecified, it could be set by default, such as to the resource
 type, like job, deployment, or replicationController. The value would need to be
 at least spatially unique, and perhaps temporally unique in the case of job.
 
-Annotations have very different intended usage from labels. We expect them to be
-primarily generated and consumed by tooling and system extensions. I'm inclined
-to generalize annotations to permit them to directly store arbitrary json. Rigid
-names and name prefixes make sense, since they are analogous to API fields.
-
-In fact, in-development API fields, including those used to represent fields of
-newer alpha/beta API versions in the older stable storage version, may be
-represented as annotations with the form `something.alpha.kubernetes.io/name` or
-`something.beta.kubernetes.io/name` (depending on our confidence in it). For
-example `net.alpha.kubernetes.io/policy` might represent an experimental network
-policy field. The "name" portion of the annotation should follow the below
-conventions for annotations. When an annotation gets promoted to a field, the
-name transformation should then be mechanical: `foo-bar` becomes `fooBar`.
+Annotations have very different intended usage from labels. They are
+primarily generated and consumed by tooling and system extensions, or are used
+by end-users to engage non-standard behavior of components.  For example, an
```

Comment:
"engage non-standard behavior of components" reads strangely, consider rephrasing